packages/mesh-extras/src/SimpleMesh.ts

```diff
@@ -24,7 +24,7 @@ export class SimpleMesh extends Mesh
      * @param {number} [drawMode] - the drawMode, can be any of the Mesh.DRAW_MODES consts
      */
     constructor(
-        texture = Texture.EMPTY,
+        texture: Texture = Texture.EMPTY,
```

Comment:
This change was required, because type inference was being used. Which, IMO, is more of a case for not inferring in situations like this (params).